README.md

```diff
@@ -7,10 +7,37 @@ Please fork this repository to your own account, either directly edit the README
 Your statements should include:
 
 * The problem you are addressing
+  Employees' work efficiency is essentil for any organization's productivity. A systematic and well organized training program prepare employees with smooth knowledge transfer and high workplace performance. Many corporates have implemented training procedures and programs allowing employees to be familiar with companies' objectives and cultures, to obtain knowledge and advance skills. However, different individual has different learning styles and ways of obsorbing information, a one-style-fit-all training session cannot achieve the optimal outcomes. 
+
 * The eductional goal that you plan to address
+  My goal is to personalize training materials through analyzing workplace data such as employee's learning curve and behavioral pattern for the purpose of enhancing corporate's productivity. 
+
 * Your objectives for the project
+  Find relevant training data  
+  Get consent to access data such as training content and employee's log file
+  Understand the purpose and end-goal of training session from both organization and employees' perspective and attitudes through interviews and surveys
+  Brainstorm factors and features that might impact the effectiveness of training material
+  Analyze the log file data
+  Come up with conclusions and recommendations on how to personalize training material
+
 * Your priorities for the project
+  Find data and gett access
+  Learning data mining skills for feature generation and analysis
+
 * What you plan to measure to achieve your educational goal and address your problem
+  * Survey:  
+  employee's competence assessment; feedback;
+  * Training material: 
+  formal and informal learning; demonstration; training methodology
+  * Data mining skills: 
+  personal informatics; device usage; web log mining; digital traces; clustering analysis; deep and surface approach to studying;  engagement; eye-tracking;  grading; interactive pattern; learning process; motivation; navigation support; problem-based learning; project-based learning; quantitative and qualitative progress; reading analytics/ reading monitor; reflective learning; self-directed learning/ self-regulated learning; self-tracking; sentiment analysis; sarcasm detection; threaded discussion; time tracking; video interaction/ annotation 
+
     * **Including an fake data set**
+    * Name  Log file ID  Minutes  Login time  Navigation support  Device usage  Sentiment analysis 
```

Comment:
Looks good. Is your intention to allocate training modality (computer, phone) based on performance metric?